courses/urls.py

```diff
@@ -6,7 +6,8 @@
 urlpatterns = [
     path('<int:course_id>/details', views.view_course, name="view_course"),
     path('<int:course_id>/modules', views.module_list, name="module_list"),
-    path('<int:course_id>/<int:module_id>/components', views.load_components, name="load_components"),
+    path('<int:course_id>/modules/<int:module_id>/components', views.load_components, name="load_components"),
+    path('quiz/<int:quiz_id>/', views.take_quiz, name="take_quiz"),
```

Comment:
Should we start this url wit hthe <int:course_id> too, to be consistent with the other URLs for the course (since this url will already start with /course/)